### ISSUE TITLE
[Feature Request?] Right-click functionality in tables

### ISSUE THREAD

**Issue by OLCadlywydd** [zadam/trilium#2126]
Would it be possible to add right-click functionality for text when multiple cells are selected? When changing text properties in tables, you currently have to change each cell's text one-by-one, by selecting the text and then right-clicking to change it. This is very time-consuming if you want to change large amounts of text across different cells. When multiple cells are selected, you can change some properties, such as the cell background colour, but you are unable to change the text within those cells at all. See the below example when attempting to change cell text colour:

![nZB8TSqMna](https://user-images.githubusercontent.com/72607789/130790543-a7b39669-f587-4e26-b8b6-7139a69147bb.gif)

**Comment by OLCadlywydd** [comment]
> 
> 
> This is a functionality of CKEditor and would have to be implemented there.

Thanks for the response @zadam. It seems like this has been implemented to [CKEditor 5 v18.0.0 as shown here](https://ckeditor.com/blog/CKEditor-5-v18.0.0-with-custom-table-selection-and-pasting-nested-lists-from-Word/). Is it possible to implement this in Trilium? Many thanks!!

![table-selection](https://user-images.githubusercontent.com/72607789/132203891-cf13f168-a9f0-43bd-9188-a547f1136d50.gif)

**Comment by zadam** [comment]
That is strange, this should be available then. Trilium integrates CKEditor 27.0 and should not have any diverging behavior in this regard.

**Comment by OLCadlywydd** [comment]
@zadam I believe that the `TableSelection` plugin needs to be enabled for this to work? https://ckeditor.com/docs/ckeditor5/latest/api/module_table_tableselection-TableSelection.html

From https://ckeditor.com/docs/ckeditor5/latest/features/table.html:
> The TableSelection plugin introduces support for the custom selection system for tables that lets you:
> 
> - Select an arbitrary rectangular table fragment — a few cells from different rows, a column (or a few of them) or a row (or multiple rows).
> - Apply formatting or add a link to all selected cells at once.
> 
> The table selection plugin is loaded automatically by the Table plugin and can be tested in the demos above.

Is it possible to add this plugin?

**Comment by zadam** [comment]
Yes, that's probably it. I wasn't aware that this is in a separate plugin. I will add this plugin in the next CKEditor upgrade (soon).
